### PR TITLE
Style: enrollment success

### LINKED ISSUE #1039: Enrollment Success page: Final touches
Make the page look like the Figma mocks. 

1. Desktop: Design the last line to this. It's a 10-column wide large div that is left-aligned.

image

image

2. Mobile: Design this last line. This mobile alignment is trickier. Might have to write CSS to add a margin-left and then calculate exactly how much the amount would be, based on what is the diameter of the icon plus the margin.

image
image

3. Enrollment Success post-log out:
image
image

image

### PR DESCRIPTION
**Pull request by angela-tran**
Closes #1039

### CODE REVIEW

**Comment by machikoyasuda** [comment]
Enrollment Success on CC Flow looks good!
image

File: benefits/static/css/styles.css [hunk 1]
```diff
@@ -538,6 +538,18 @@ h1 + .media-list, /* A .media-list immediately following the h1: Enrollment, Enr
 /* Enrollment Success */
 /* Custom in-line log out button for Enrollment Success, in paragraph */
 
+:root {
+  --logout-paragraph-margin-left: calc(
+    var(--media-item-icon-margin) + var(--media-item-icon-size)
```
Comment (machikoyasuda) [inline benefits/static/css/styles.css:543]: 🧮 🎉

Comment (angela-tran) [reply benefits/static/css/styles.css:543]: Thank you for your help on that!! 🧮

**Comment by angela-tran** [comment]
Working on the logged_out page now

**Comment by angela-tran** [comment]
I'm not sure how we want to achieve this design where 
 - mobile has some spacing between the two sentences in the H1
 - desktop has no spacing

 vs.

**Comment by angela-tran** [comment]
Re: https://github.com/cal-itp/benefits/pull/1056#issuecomment-1276914395
@machikoyasuda I think maybe we could have two .PO entries, one for each sentence. They would be within one H1 element, and for the spacing in between, maybe we could have a `span` or something that has mobile-only styles? I'm not sure exactly what those styles would be.

What do you think? Do you have other ideas on how to do this?

**Comment by machikoyasuda** [comment]
> Re: [#1056 (comment)](https://github.com/cal-itp/benefits/pull/1056#issuecomment-1276914395) @machikoyasuda I think maybe we could have two .PO entries, one for each sentence. They would be within one H1 element, and for the spacing in between, maybe we could have a `span` or something that has mobile-only styles? I'm not sure exactly what those styles would be.
> 
> What do you think? Do you have other ideas on how to do this?

```html
    <h1 class="pb-lg-8 pb-4">
      <span class="h2 d-block d-lg-inline">You have successfully logged out.</span>
      <span class="h2 d-block d-lg-inline">Thank you for using Cal-ITP Benefits!</span>
    </h1>
```

image
image

Technically the `d-block d-lg-inline` on the second <span> is not necessary though. Could just do the 1st one.

File: benefits/core/templates/core/includes/icon.html [hunk 2]
```diff
@@ -9,5 +9,5 @@
   </p>
 {% else %}
   {% load static %}
-  <img class="icon" src="{% static icon.src %}" alt="{{ icon.alt }}" />
+  <img class="icon" width="150px" height="150px" src="{% static icon.src %}" alt="{{ icon.alt }}" />
```
Comment (machikoyasuda) [inline benefits/core/templates/core/includes/icon.html:12]: Don't need the px here

Comment (angela-tran) [reply benefits/core/templates/core/includes/icon.html]: Edited my commit to remove `px` from these two attribute values

**Review by machikoyasuda — APPROVED** [review]
Tested Login.gov flow, CC flow locally on Desktop and Mobile widths.

**Comment by machikoyasuda** [comment]
From now on: Whenever we make changes to `base.html` we should also double-check we aren't causing any regressions on any pages that use `base.html`.

**Pull request merged**